<map id="django.contrib.gis.gdal.prototypes.errcheck.check_geom" name="django.contrib.gis.gdal.prototypes.errcheck.check_geom">
<area shape="rect" id="node2" href="$d6/d64/namespacedjango_1_1contrib_1_1gis_1_1gdal_1_1prototypes_1_1errcheck.html#a64cc865cda739282789d6fc678e6ba93" title="django.contrib.gis.gdal.prototypes.errcheck.check\l_geom_offset" alt="" coords="403,5,713,47"/>
<area shape="rect" id="node3" href="$d5/d9a/namespacedjango_1_1contrib_1_1gis_1_1gdal_1_1prototypes_1_1generation.html#aced3f532898a49c4a34395716708a6bd" title="django.contrib.gis.gdal.prototypes.generation.geom\l_output" alt="" coords="761,5,1079,47"/>
</map>
